examples/assembly/Add_Array_(AArch64).asm

```diff
@@ -0,0 +1,191 @@
+# This file demonstrates adding 2 arrays using serial, SVE and NEON instructions.
+# Compile with aarch64-linux-gnu-gcc -mcpu=generic+sve (or -march=armv8.2-a+sve),
```

Comment:
This is ok but we need to say what the user in compiler explorer needs to select as well, first.

Like the Intel example has:
```
; Assemble with NASM and -f elf64
```

Giving the local compiler command is useful to for updating this but ultimately the compiler explorer settings are what the reader wants up front.

Reply:
Like this:
https://godbolt.org/z/xfP7zs56G

```
# This file demonstrates adding 2 arrays using serial, SVE and NEON instructions.

# To assemble in Compiler Explorer select AArch64 binutils and add
# "-march=armv8.2-a+sve" to the compiler options.

# To compile locally use "aarch64-linux-gnu-gcc -mcpu=generic+sve" (or -march=armv8.2-a+sve),
# the compiler driver will set up the call main.
```